### PR TITLE
Add argparse and confuse

### LINKED ISSUE #97: Add argparse and confuse

### PR DESCRIPTION
**Pull request by IKostric**
Fixes #97

### CODE REVIEW

File: moviebot/run.py [hunk 1]
```diff
@@ -17,101 +28,47 @@
...
-                return "ValueErrorType:{}".format(details[-1])
-        else:
-            return "FileNotFoundError"
-    else:
-        return "ValueError"
-
-
-def arg_parse(args=None):
-    """Parses the arguments in the configuration file
+def parse_args(args=None) -> argparse.Namespace:
```
Comment (vinaysetty) [inline moviebot/run.py:31]: Type hint for args is missing.

File: moviebot/run.py [hunk 2]
```diff
@@ -17,101 +28,47 @@
...
+    """Parse command line arguments.
 
     Args:
-      args: configuration file for the dialogue (Default value = None)
+        args (optional): List of arguments to parse. If not provided, uses
+            sys.argv[1:]. Defaults to None.
```
Comment (vinaysetty) [inline moviebot/run.py:36]: sys.argv[1:] is not a param?

**Review by vinaysetty — APPROVED** [review]
LGTM!

File: moviebot/run.py (hunk 2 above)
Comment (IKostric) [reply moviebot/run.py:36]: ArgumentParser defaults to `sys.argv[1:]`.

File: moviebot/run.py (hunk 1 above)
Comment (IKostric) [reply moviebot/run.py:31]: Done

**Pull request merged**